src/components/Editor/FormikEditor.jsx

```diff
@@ -25,9 +26,15 @@ const FormikEditor = (
     const prevFormikProps = prevProps.formik;
     const nextFormikProps = nextProps.formik;
 
+    const pathParts = name.split(".");
+    const prevError = path(pathParts, prevFormikProps.errors);
+    const nextError = path(pathParts, nextFormikProps.errors);
+    const prevTouched = path(pathParts, prevFormikProps.touched);
+    const nextTouched = path(pathParts, nextFormikProps.touched);
```

Comment:
@AbhayVAshokan Please let me know if we have a better way of implementing this with a compact function. I couldn't find one in neeto-cist. Found https://lodash.com/docs/4.17.15#get, however decided to stick with Ramda respecting immutability.

Reply:
This looks neat. Just what I had in mind.